pulsar-client-admin-api/src/main/java/org/apache/pulsar/client/admin/Topics.java

```diff
@@ -249,6 +250,49 @@ CompletableFuture<List<String>> getListAsync(String namespace, TopicDomain topic
      */
     CompletableFuture<List<String>> getPartitionedTopicListAsync(String namespace);
 
+    /**
+     * Get the list of partitioned topics under a namespace.
+     * <p/>
+     * Response example:
+     *
+     * <pre>
+     * <code>["persistent://my-tenant/my-namespace/topic-1",
+     *  "persistent://my-tenant/my-namespace/topic-2"]</code>
+     * </pre>
+     *
+     * @param namespace
+     *            Namespace name
+     * @param params
+     *            params to filter the results
+     * @return a list of partitioned topics
+     *
+     * @throws NotAuthorizedException
+     *             Don't have admin permission
+     * @throws NotFoundException
+     *             Namespace does not exist
+     * @throws PulsarAdminException
+     *             Unexpected error
+     */
+    List<String> getPartitionedTopicList(String namespace, Map<QueryParam, Object> params) throws PulsarAdminException;
```

Comment:
We'd better use a `ListTopicsOptions` to keep consistent with the 

```
 TopicStats getStats(String topic, GetStatsOptions getStatsOptions) throws PulsarAdminException;
```

The `QueryParam` is not easy to use, through the API, we still don't know the value type.

Reply:
fixed.